### DISCUSSION TITLE
"allowFreeFlowerPlacing" / "allowFreeBlockPlacing"

### DISCUSSION THREAD

**Discussion by ghost** [Minecraft-AMS/Carpet-AMS-Addition#75] [Ideas]
Being able to place blocks where it was before not possible.
>>>
Flowers can be placed everywhere. And while you remove one block, it creates no block updates and removes surrounded blocks.

I use a script from Carpet, but a optimized rule sounds better and could solve the problem with the block updates.
Example: I place 4 flowers. Every flower is surrounded to each other, if I break now one flower, it drops all flowers.

Includes Flowers, Saplings, Mushrooms - maybe there are more candidates.

**Comment by wendavid552** [comment]
It is interesting, and useful for some Minecraft builders. With a totally vanilla method, you would need an update surpressor. I would rather consider it as a CREATIVE rules, which only restrict the neighbour updates from flowers. 

By the way, there is a way that disable interation update now in Carpet-AMS-Addition, that is to use the `/interation` command.

**Reply by ghost** [reply]
All my ideas are for Vanilla survival mode in mind. But I think with Creative rules you mean the category, right?

Interesting. Let's see how you decide about this idea 😊
